lib/tls.js

```diff
@@ -231,7 +231,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 // Example:
 // C=US\nST=CA\nL=SF\nO=Joyent\nOU=Node.js\nCN=ca1\nemailAddress=ry@clouds.org
 exports.parseCertString = function parseCertString(s) {
-  var out = {};
+  var out = Object.create(null);
```

Comment:
It might, hard to say.  It would break `conn.getPeerCertificate().hasOwnProperty(k)` but then again, such code is vulnerable to object manipulation now.